content/platform/zerto/home.md

```diff
@@ -1,34 +1,46 @@
 
 Learn more about [Zerto](https://www.hpe.com/us/en/zerto.html)
 
 ## Welcome to the Zerto Hacker Hub!
 
-Here you will be provided the tools to get started in deploying and managing Zerto’s continuous data protection solution that leverages its proprietary near-synchronous replication capabilities to protect and restore virtual machines, Kubernetes clusters, and public cloud instances. Zerto offers an extensive REST API with Swagger support, for users to programmatically create and manage end-to-end disaster recovery workflows.  
+Here you will be provided the tools to get started in deploying and managing Zerto’s CDP that leverages its proprietary near-synchronous replication capabilities to protect and restore virtual machines and public cloud instances. Zerto offers an extensive REST API with Swagger support, for users to programmatically create and manage end-to-end disaster recovery workflows.  
 
 Get started by diving into [our GitHub page](https://github.com/ZertoPublic) or check out the documentation at <https://help.zerto.com>
 
-### Zerto for Kubernetes
+### Simplify Automation with Zerto’s REST Swagger APIs
 
-Zerto for Kubernetes (Z4K) enables continuous backup, disaster recovery, and long-term retention for stateful and stateless Kubernetes applications running on-premises or in the cloud.
+Zerto’s REST Swagger APIs offer a powerful and user-friendly way to automate and integrate Zerto into your existing IT workflows. These APIs provide a comprehensive set of endpoints that allow you to programmatically manage, monitor, and control your Zerto environment with ease. Whether you are looking to automate routine tasks, integrate with third-party tools, or develop custom applications, Zerto’s REST APIs make it straightforward and efficient.
 
-Learn more by watching Zerto for Kubernetes Overview 
+With Swagger, developers can interactively explore and test the APIs, making it easier to understand their capabilities and integrate them into your systems. The Swagger interface provides clear documentation and real-time testing capabilities, ensuring that you can quickly get up to speed and start leveraging Zerto’s powerful features.
 
-[![Zerto for Kubernetes Overview](https://img.youtube.com/vi/EIQcOIcbBwU/hqdefault.jpg)](https://www.youtube.com/watch?v=EIQcOIcbBwU)
+Unlock the full potential of your disaster recovery strategy with Zerto’s REST Swagger APIs today!
+
+For more information and to access the Zerto REST Swagger APIs, visit the [Zerto API Documentation](https://www.zerto.com/myzerto/apis/).
+
+
+
+### Unlock Powerful Insights with Zerto Resiliency Observation Console
```

Comment:
Per Brand requirements, change to:
"Unlock powerful insights with Zerto Resiliency Observation Console"